### ISSUE TITLE
RestoreIot unsupported product

### ISSUE THREAD

**Issue by evbarnett5** [dgreif/homebridge-hatch-baby-rest#88]
Just bought  a new hatch restore running firmware (9.1.361) . Installed plugin via home bridge UI and is currently logging that restoreIot is an unknown product.
<img width="1085" alt="PastedGraphic-1" src="https://user-images.githubusercontent.com/115050950/193955092-5e875008-bf7b-4b13-b8fb-5595a6354e5f.png">

**Comment by evbarnett5** [comment]
TLDR: Modified scripts. Hatch status works in homekit when turned on from hatch app. Could not control hatch from homekit. Device is currently stuck in a connection loop and awaiting support from hatch. 

Wanted to provide an update on this. Modified some of the scripts to handle the newly named restore device (restoreIot). Got to the point where if I turned on the device from the hatch app the homekit accessory would update with the correct status. However, I couldn't get the restore to turn on using the homekit switch. I was debugging some of the MQTT messages I saw when I would control from the app and tried to modify the structure that homekit sends to turn on the device. This caused the device to get stuck in some weird connection loop where the app thinks the device isn't connected and therefore won't propagate any changes made in the app. Reached out to Hatch to see if they can reset something from their side and currently awaiting a reply. Will provide additional updates as I find things out. Happy to share changes I have made thus far.
